assets/js/atomic/blocks/product-elements/rating/attributes.ts

```diff
@@ -1,4 +1,4 @@
-export const blockAttributes = {
+export const blockAttributes: Record< string, Record< string, unknown > > = {
```

Comment:
how about using `BlockAttributes` from `@wordpress/blocks` here?

Reply:
> how about using `BlockAttributes` from `@wordpress/blocks` here?

Technically, we could use `BlockAttributes` from `@wordpress/blocks`. However, I feel that that definition is very loose:
```ts
export type BlockAttributes = Record<string, any>;
```

While `Record< string, Record< string, unknown > >` is not perfect either, it's more typed than `BlockAttributes` from `@wordpress/blocks`. That's why I'd rather use the initial one. Would you agree to that, @opr?

Reply:
Hi Niels, I'm sorry for the late reply, I missed the ping.

My thoughts are that if it's good enough for Gutenberg why should we deviate from it? What benefit does it compare to know the values are also going to be objects?

Reply:
> My thoughts are that if it's good enough for Gutenberg why should we deviate from it? What benefit does it compare to know the values are also going to be objects?

Good point, @opr. To keep the other product elements aligned, I've applied this change to various files.

Would you mind reviewing this PR again?